root/usr/share/gpoint/lib/nmea.lua

```diff
@@ -168,8 +168,9 @@ local function getRMC(GnssData, resp)
 		"date",      -- Date, ddmmyy
 		"mv",        -- Magnetic Variation, degrees
 		"ewm",       -- E or W
-		"nstat"      -- Nav Status (NMEA 4.1 and later) A=autonomous, D=differential, E=Estimated, -> 
-					 -- -> M=Manual input mode N=not valid, S=Simulator, V = Valid
+		"nstat",      -- Nav Status (NMEA 4.1 and later) A=autonomous, D=differential, E=Estimated, -> 
+						-- M=Manual input mode N=not valid, S=Simulator, V = Valid
+		"sc", 			--checksum
```

Comment:
Why are the control amounts needed in this particular data? The comma after the checksum is not needed